### DISCUSSION TITLE
State value too long

### DISCUSSION THREAD

**Discussion by leeyuentuen** [john30/ebusd#547] [Q&A]
When I use ebus on home assistant. It seems the state value is too long to let HA process them:

> Exception in message_received when handling msg on 'ebusd/bai/SetMode': '{ "hcmode": {"value": "auto"}, "flowtempdesired": {"value": 28.0}, "hwctempdesired": {"value": 59.0}, "hwcflowtempdesired": {"value": null}, "disablehc": {"value": 0}, "disablehwctapping": {"value": 0}, "disablehwcload": {"value": 0}, "remoteControlHcPump": {"value": 0}, "releaseBackup": {"value": 0}, "releaseCooling": {"value": 0}}' Traceback (most recent call last): File "/usr/src/homeassistant/homeassistant/components/mqtt/debug_info.py", line 42, in wrapper msg_callback(msg) File "/usr/src/homeassistant/homeassistant/components/mqtt/sensor.py", line 282, in message_received self.async_write_ha_state() File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 539, in async_write_ha_state self._async_write_ha_state() File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 663, in _async_write_ha_state self.hass.states.async_set( File "/usr/src/homeassistant/homeassistant/core.py", line 1357, in async_set state = State( File "/usr/src/homeassistant/homeassistant/core.py", line 1045, in __init__ raise InvalidStateError( homeassistant.exceptions.InvalidStateError: Invalid state encountered for entity ID: sensor.setmode. State max length is 255 characters.

> Exception in message_received when handling msg on 'ebusd/global/updatecheck': '"revision v22.1 available, broadcast.csv: different version available, memory.csv: different version available, vaillant/08.bai.csv: different version available, vaillant/bai.308523.inc: different version available, vaillant/broadcast.csv: different version available, vaillant/errors.inc: different version available, vaillant/general.csv: different version available, vaillant/hcmode.inc: different version available, vaillant/timerhc.inc: different version available"' Traceback (most recent call last): File "/usr/src/homeassistant/homeassistant/components/mqtt/debug_info.py", line 42, in wrapper msg_callback(msg) File "/usr/src/homeassistant/homeassistant/components/mqtt/sensor.py", line 282, in message_received self.async_write_ha_state() File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 539, in async_write_ha_state self._async_write_ha_state() File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 663, in _async_write_ha_state self.hass.states.async_set( File "/usr/src/homeassistant/homeassistant/core.py", line 1357, in async_set state = State( File "/usr/src/homeassistant/homeassistant/core.py", line 1045, in __init__ raise InvalidStateError( homeassistant.exceptions.InvalidStateError: Invalid state encountered for entity ID: sensor.ebusd_updatecheck_2. State max length is 255 characters.

**Comment by john30** [comment]
duplicate of #542
